templates/premium/character_reset_complete.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * @var string $self_link
+ */
+?>
+
+<table class='table'><tr><th>Character Reset</th></tr>
+    <tr><td style='text-align:center;'>
+        You have reset your character.<br />
+        <a href='<?=$system->link?>?id=1'>Continue</a>
```

Comment:
```suggestion
        <a href='<?= $system->links['profile'] ?>'>Continue</a>
```
I think ideally we should start moving to this pattern at least for common links. This gives us flexibility